Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>You can shortcut a lot of this by using our nbinit function - we hide a lot of msticpy and common libs like pandas, numpy, seaborn behind this call. Most of the other notebooks have it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>